framework/test/bundles/libC1/TestBundleC1.cpp

```diff
@@ -138,6 +132,8 @@ namespace cppmicroservices
                         count--;
                     }
                 }
+                // stop spin
```

Comment:
its more helpful to other developers to example the "why" in a comment here instead of describing the "what". I recommend adding a short, succinct note about why a `sleep_for` in a loop is being used.